cmd/container-disk-v1alpha/BUILD.bazel

```diff
@@ -0,0 +1,125 @@
+load(
+    "@io_bazel_rules_docker//container:container.bzl",
+    "container_image",
+)
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
+
+container_image(
+    name = "container-disk-v1alpha-image",
+    base = "@libvirt//image",
```

Comment:
this is adding things to the container disk base image that isn't required  for container disks to function.